pywbemtools/pywbemcli/_cmd_connection.py

```diff
@@ -483,20 +480,11 @@ def cmd_connection_delete(context, name, options):
 
     name = select_connection(name, context, connections)
 
-    context.spinner.stop()
```

Comment:
Are you sure that `context.spinner.stop()` can be removed? In connection save, you kept it.

Reply:
Yes because just above  the get_current_connection stops the spinner. The save does not call  select_ connection so I stop the spinner before the method ends.

Reply:
Put a comment in just to avoid confusion.

Also ran makebuild again and put update in new push